test/system/005-info.bats

```diff
@@ -176,25 +176,39 @@ host.slirp4netns.executable | $expr_path
 
     containersConf=$PODMAN_TMPDIR/containers.conf
     cat >$containersConf <<EOF
-[engine]
```

Comment:
This sure got complicated in a hurry. See #20563 for explanation of why a single run of podman with `--db-backend=boltdb` will forevermore screw up the entire system.

On the plus side, I think this new test is better and more complete.